tools/reproduce_comparison/Jenkinsfile

```diff
@@ -95,45 +99,66 @@ pipeline {
+                        if (untarOriginalJDK != 0 ) {
+                            currentBuild.result = 'UNSTABLE'
+                            error " Unzip ${originalJDKFile[0].name} failed"
+                            return
+                        }
+                        def untarReproducedJDK = sh returnStatus: true, script: "cd reproduced && temp=`mktemp -d` && unzip -d ${temp} ${reproducedJDKFile[0].name} && mv ${temp}/*/* ./ && rmdir ${temp}/* ${temp}"
+                        if (untarReproducedJDK != 0 ) {
+                            currentBuild.result = 'UNSTABLE'
+                            error " Unzip ${originalJDKFile[0].name} failed"
+                            return
+                        }
+                        sh "find . -type f -name 'OpenJDK*-jdk*.zip' -delete"
+                    } else {
+                        def originalJDKFile = findFiles(glob: "original/*.tar.gz")
+                        def reproducedJDKFile = findFiles(glob: "reproduced/*.tar.gz")
+                        def untarOriginalJDK = sh returnStatus: true, script: "tar xzf ${originalJDKFile[0].path} --strip-components=1 -C original"
+                        if (untarOriginalJDK != 0 ) {
+                            currentBuild.result = 'UNSTABLE'
+                            error " Untar ${originalJDKFile[0].name} failed"
+                            return
+                        }
+                        def untarReproducedJDK = sh returnStatus: true, script: "tar xzf ${reproducedJDKFile[0].path} --strip-components=1 -C reproduced"
+                        if (untarReproducedJDK != 0 ) {
+                            currentBuild.result = 'UNSTABLE'
+                            error " Untar ${reproducedJDKFile[0].name} failed"
+                            return
+                        }
+                        sh "find . -type f -name 'OpenJDK*-jdk*.tar.gz' -delete"
+                    }
                     try {
                         dir('temurin-build/tooling') {
                             def rc = 0
                             if (COMPARED_JOB_NAME.contains('linux')) {
-                                rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced"
+                                rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced Linux"
                             } else if (COMPARED_JOB_NAME.contains('mac')) {
                                 // mac
-                                //sh "./tooling/mac_repro_compare.sh temurin ./original temurin ./reproduced ${self_cert_file} ${self_cert_passwd}"
+                                //rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced Darwin"
```

Comment:
commented out because it's not supported yet?

Reply:
correct, we will work on mac later.